tests/orm/test_comments.py

```diff
@@ -194,3 +195,11 @@ def test_objects_get(self):
         comment = node.add_comment('Check out the comment on _this_ one')
         gotten_comment = Comment.objects.get(id=comment.id)
         assert isinstance(gotten_comment, Comment)
+
+    def test_delete_node_with_comments(self):
+        """Test deleting a node with comments."""
+        node = orm.Data().store()
+        node.add_comment('Check out the comment on _this_ one')
+        assert len(Comment.objects.all()) == 2
```

Comment:
This seems fragile, relying on the fact that one comment already exists. Would be better to clean the database first or change the check to directly check for the existence of the explicit comment created.

Reply:
The database is cleaned before each test, then a node and comment is added (in ` def init_profile`, this fixture is run before every test method in the `TestComment` class).
But anyhow, I've just removed the creation of an additional node/comment, and check for the deletion of the one created in `init_profile`